src/shadowbox/docker/run_action.sh

```diff
@@ -17,7 +17,7 @@
 do_action shadowbox/docker/build
 
 readonly OUTLINE_DIR=/tmp/outline
-readonly STATE_DIR=$OUTLINE_DIR/persisted-state
+readonly HOST_STATE_DIR=$OUTLINE_DIR/persisted-state
 readonly CONTAINER_STATE_DIR=/root/shadowbox/persisted-state
 readonly STATE_CONFIG=$STATE_DIR/shadowbox_server_config.json
```

Comment:
You need to update all references to the old STATE_DIR

Reply:
Done